docs/virtual-env.md

```diff
@@ -2,9 +2,8 @@
 
 ## Dependencies
 
-- [Virtualenv](http://www.virtualenv.org/en/latest/)
 - [Python 3](http://www.python.org/) (Can be installed using `brew install python3`)
-- [nodejs.org](http://nodejs.org/) (v8.12 - can be installed using [nvm](https://github.com/creationix/nvm))
+- [nodejs.org](http://nodejs.org/) (v10.24 - can be installed using [nvm](https://github.com/creationix/nvm))
```

Comment:
How about we make this easier for people to do this with nvm, by being more explicit? e.g. https://github.com/ministryofjustice/cla_frontend/blob/master/README.md#nodejs-v89x